backend/clients/twilio_client.py

```diff
@@ -70,7 +77,21 @@ def save_chat_history(
         )
         session.add(new_chat)
         session.commit()
+
+        # handle media
+        if media:
+            for md in media:
+                new_media = Chat_Media(
+                    chat_id=new_chat.id,
+                    url=md.get('url'),
+                    type=md.get('type')
+                )
+                session.add(new_media)
+                session.commit()
+        # eol handle media
```

Comment:
Small performance issue: You could move the `session.commit()` from line 79 to here and remove the `commit()` on line 90. That way you only have a single commit for each message, rather than having one commit per media file, plus another for the message itself.

For our purposes this is not a big deal, as we don't have the volume.